index.js

```diff
@@ -45,10 +45,12 @@ Use the copy function below to do the following:
     2. Return a copy of the received array  
 */
 
-function copy(/*your code here*/){
-    /*your code here*/
+function copy(originalFlavors){
+    const flavorsCopy = [...originalFlavors];
```

Comment:
Great variable naming.  flavorsCopy is a great name which conveys exactly what it stands for.  Excellent use of the spread operator.  Let me know if you have any questions about it.